src/Razor/src/Microsoft.CodeAnalysis.Razor/BindTagHelperDescriptorProvider.cs

```diff
@@ -207,7 +207,9 @@ private List<ElementBindData> GetElementBindData(Compilation compilation)
                 {
                     var attribute = attributes[j];
 
-                    if (attribute.AttributeClass == bindElement)
+                    // We need to check the constructor argument length here, because this can show up as 0
+                    // if the language service fails to initialize. This is an invalid case, so skip it.
+                    if (attribute.AttributeClass == bindElement && attribute.ConstructorArguments.Length == 4)
```

Comment:
Do we ever envision adding more constructor args? Might be restrictive to enforce this to 4, `>= 4`?

Reply:
If that happens this code will change no? This seems like something that would be obvious when we revise this.